### DISCUSSION TITLE
Can I serve swagger docs from an existing openapi.yml file?

### DISCUSSION THREAD

**Discussion by geoff-maddock** [DarkaOnLine/L5-Swagger#519] [Q&A]
I'm looking for a good way to generate swagger web docs page for my API project.  
I created by API by first creating a .yaml spec, and then implementing it in my project - so I didn't use the annotations here.

Is it possible to point this project to my .yaml file and have it generate the /docs url?

**Comment by DarkaOnLine** [comment]
Yes you can. You should change this path to point to your file: https://github.com/DarkaOnLine/L5-Swagger/blob/master/config/l5-swagger.php#L31 and change this flag - https://github.com/DarkaOnLine/L5-Swagger/blob/master/config/l5-swagger.php#L36

**Reply by geoff-maddock** [reply]
Seems like one limitation is that the file needs to be publicly accessable?  I tried to provide a path to the file which is not public, but it treated it like it was a subfolder of the public folder.
